src/pages/Assignment.tsx

```diff
@@ -162,7 +162,7 @@ const AssignmentPage: React.FC = () => {
                       lessonsScoreMap={lessonResultMap || {}}
                       startIndex={0}
                       showSubjectName={true}
-                      showChapterName= {true}
+                      showChapterName= {false}
```

Comment:
Why you made showChapterName= {false}. i feel we need to show chapter name in assignment page right